opentelemetry-api/src/opentelemetry/context/unified_context.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.distributedcontext import DistributedContext
+from opentelemetry.trace import SpanContext
+
+
+class UnifiedContext:
```

Comment:
* this is typically called just `Context` (e.g. io.grpc.Context in Java, `context.Context` in Go).
* by creating a dependency on DistributedContext and SpanContext this is creating the inverted relationship, violating the layering principle described in https://docs.google.com/document/d/1UxrEYOaQlF_E4gtiPoFmcZ4YKKe1GxohvCvQDuwvD1I/

Reply:
this module could easily move out of the "Context" module. I think I just added that because the module name seemed to match.

In reality I imagine this would be higher-level container data structure, in a separate module.